assets/files.wxs

```diff
@@ -3065,6 +3065,23 @@
       <Component Id="cmpEC42F4F98D70437BB0E290A0E0FE1775" Guid="{5483092a-12c2-4e6c-b730-be491b8909c0}">
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-macos`
Group policy settings tests.Group policy settings tests.Transcription policy test
```powershell
Cannot bind argument to parameter 'Path' because it is null.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/Basic/GroupPolicySettings.Tests.ps1: line 170
170:             if (-not (Test-Path $KeyPath)) {$null = New-Item $KeyPath}

```